packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -98,6 +98,47 @@ export interface AwsSdkCall {
   readonly outputPath?: string;
 }
 
+/**
+ * The IAM Policy that will be applied to the different calls.
+ */
+export class AwsCustomResourcePolicy {
+
+  /**
+   * Use this constant to configure access to ALL resources.
+   */
+  public static readonly ALL_RESOURCES = ['*'];
```

Comment:
I am wondering if this should be `ANY_RESOURCE`

Reply:
Yes, thats better.